crates/bevy_ui/src/focus.rs

```diff
@@ -53,6 +53,27 @@ impl Default for Interaction {
     }
 }
 
+/// Describes whether [`Interaction`] component should remain in the `Clicked` state after
```

Comment:
These docs need to be clearer. It's hard to understand exactly what behavior is being performed here, and what the default behavior is.

Reply:
Is it alright now?